.github/workflows/integration-test.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       matrix:
         # TODO: readd macos-latest once we fix the integration tests
         os: [ ubuntu-latest, windows-latest ]
-        python-version: [ 3.7, 3.8, 3.9 ]
+        python-version: [3.7, 3.8, 3.9, '3.10', '3.11']
```

Comment:
oh cool, you got it working!

Reply:
Unfortunately not yet, GH actions just decided to not run for this commit, I think they're having issues today as I've had that a few times

Reply:
Had to remove this again due to some integrations (tensorflow, pytorch and probably others) don't support Python 3.11 yet.